src/bindings/cpp/examples/cpp_example_hierarchy.cpp

```diff
@@ -70,7 +70,7 @@ unsigned long name_depth (Key k)
 class Visitor
 {
 public:
-	virtual void visit (std::string name, unsigned long depth, Key k) = 0;
+	virtual void visit (std::string const & name, unsigned long depth, Key k) = 0;
```

Comment:
Exactly this "in general" is not true anymore. So better not blindly replace the one with the other. You can keep the current changes if it is work to undo them.